docs/source/guide/webhook_reference.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Webhook event format reference 
+short: Webhooks Event Reference
+type: guide
+order: 653
+meta_title: Label Studio Webhook Event Reference 
+meta_description: Label Studio reference documentation for webhook event fields and payloads sent from Label Studio for integration with your machine learning pipeline. 
+---
+
+Label Studio includes several types of webhook events that trigger when specific actions occur. Refer to the details on this page to determine what information is included in the payload of each webhook event. For details about how to use webhooks and to set up webhooks in Label Studio, see [Set up webhooks in Label Studio](webhooks.html). To create your own webhooks in Label Studio to trigger events when specific actions happen, see [Create custom events for webhooks in Label Studio](webhook_create.html). 
+
+Webhooks sent from Label Studio include the following:
+
+| Key | Details |
+| --- | --- |
+| action | Details the action that the event represents. |
+| project | Included only for task and annotation events. Details about the project. |
```

Comment:
do we need to mention main body like `task` included here as well?

Reply:
is that sent for all webhooks or just some of them?